ramls/oai-pmh-view.raml

```diff
@@ -20,14 +21,14 @@ resourceTypes:
   collection-get: !include raml-util/rtypes/collection-get.raml
 
 /oai-pmh-view:
-  /instances:
-    displayName: Stream API to get data for OAI-PMH
+  /updatedInstanceIds:
+    displayName: Stream API to get instances ids for OAI-PMH
```

Comment:
Was it considered that the existing API could have remained in place, and the updated ID specific variant added as a new endpoint, in order to avoid the break in compatibility?